Cython/Compiler/FusedNode.py

```diff
@@ -458,6 +483,12 @@ def _buffer_declarations(self, pyx_code, decl_code, all_buffer_types):
                 except (ImportError, AttributeError, TypeError):
                     ndarray = None
             """)
+        pyx_code.func_defs.put_chunk(
+            u"""
+                def __is_little_endian():
+                    cdef int endian_detector = 1
+                    return (<char*>&endian_detector)[0] != 0
```

Comment:
Fixed in https://github.com/cython/cython/pull/1607/commits/e3fb29c91b13edcc9297b02b0ca10e9a3da3928d